org.adempiere.base/src/org/compiere/model/MStorageOnHand.java

```diff
@@ -242,7 +242,55 @@ public static MStorageOnHand[] getAll (Properties ctx,
 		list.toArray(retValue);
 		return retValue;
 	}	//	getAll
-
+	
+	/**
+	 * 	Get Storage Info
+	 *	@param ctx context
+	 *	@param M_Locator_ID locator
+	 *	@param M_Product_ID product
+	 *	@param M_AttributeSetInstance_ID instance
+	 *  @param dateMPolicy
+	 *  @param ignoreZeroQty
+	 *	@param trxName transaction
+	 *	@return existing or null
+	 */
+	public static MStorageOnHand[] getAll (Properties ctx, 
+		int M_Locator_ID, int M_Product_ID, int M_AttributeSetInstance_ID, Timestamp dateMPolicy, boolean ignoreZeroQty, String trxName)
```

Comment:
in all other place, the M_Product_ID parameter comes before M_Locator_ID, better follows that convention here as well.